docs/source/examples/ggi_summer_school_solved.md

```diff
@@ -0,0 +1,313 @@
+---
+orphan: true
+
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.12.0
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Solution to hands-on lab on error mitigation with Mitiq.
++++
+
+This is a hands-on notebook created for the [`SQMS/GGI 2022 Summer School on Quantum Simulation of Field Theories`](https://www.ggi.infn.it/showevent.pl?id=436). 
+
+It is a guided tutorial on error mitigation with Mitiq and is focused on the zero-noise extrapolation (ZNE) technique. As this is
+intended to be a hands-on exercise, the solutions to the examples are provided in this notebook.  
+
+Useful links :
+
+- [`Mitiq repository`](https://github.com/unitaryfund/mitiq)
+- [`Mitiq documentation`](https://mitiq.readthedocs.io/en/stable/)
+- [`Mitiq docs on ZNE`](https://mitiq.readthedocs.io/en/stable/guide/zne.html)
+- [`Mitiq white paper`](https://arxiv.org/abs/2009.04417)
+- [`Unitary Fund`](https://unitary.fund)
+
+```{figure} ../img/zne_workflow2_steps.png
+---
+width: 400
+name: figzne1
+---
+The diagram shows the workflow of the zero noise extrapolation (ZNE) technique in Mitiq.
+```
+
+The lab is split into the following sections :
+
+- [](#checking-python-packages-are-installed-correctly)
+- [](#computing-a-quantum-expectation-value-without-error-mitigation)
+- [](#apply-zero-noise-extrapolation-with-mitiq)
+- [](#explicitly-selecting-the-noise-scaling-method-and-the-extrapolation-method)
+- [](#what-happens-behind-the-scenes-a-low-level-application-of-zne)
++++
+
+## Checking Python packages are installed correctly
+
+This notebook was tested with **Mitiq v0.20.0** and **qiskit v0.39.0**. It probably works with other versions too. Moreover, with minor changes, it can be adapted to quantum libraries that are different from Qiskit: Cirq, Braket, PyQuil, etc..
```

Comment:
```suggestion
This notebook was tested with **Mitiq v0.23.0** and **qiskit v0.41.0**. It probably works with other versions too. Moreover, with minor changes, it can be adapted to quantum libraries that are different from Qiskit: Cirq, Braket, PyQuil, etc..
```